src/components/dialog/bl-dialog.css

```diff
@@ -1,36 +1,52 @@
-dialog {
+.dialog,
+.dialog-polyfill .container {
   padding: 0;
   border: 0;
   background: var(--bl-color-primary-background);
   border-radius: var(--bl-border-radius-l);
   max-width: calc(100vw - var(--bl-size-4xl));
   max-height: calc(100vh - var(--bl-size-4xl));
+  min-width: 424px;
+  min-height: 178px;
   overflow: hidden;
 }
 
-dialog::backdrop {
+.dialog::backdrop {
   background: #273142;
   opacity: 0.7;
 }
 
-dialog .container {
-  min-width: 424px;
-  min-height: 178px;
+.dialog-polyfill {
+  display: var(--display);
```

Comment:
Use `[open]` selector instead of variable. Check below for the detailed comment.